src/util/psbt/map/global.rs

```diff
@@ -155,6 +250,55 @@ impl Decodable for Global {
                                 return Err(Error::InvalidKey(pair.key).into())
                             }
                         }
+                        PSBT_GLOBAL_KEY_XPUB => {
+                            if !pair.key.key.is_empty() {
+                                let xpub = ExtendedPubKey::decode(&pair.key.key)
+                                    .map_err(|_| {
+                                        encode::Error::ParseFailed("Can't deserialize ExtendedPublicKey from global XPUB key data")
```

Comment:
nit: This exceeds 100 characters, right? how about putting the encode::Error::ParseFailed on the same line as `.map_err`?